### ISSUE TITLE
Sniff names too long

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#296]
When running phpcs with reporting to file turned on, like so:
```bash
phpcs -s . --standard=./ruleset.xml --extensions=php --report-full=./phpcs.txt --report-source=./phpcs-source.txt
```

I get the following PHP error which is caused by the default report-width being 80 and the sniff name for this WP sniff being 74 chars which combined with the count and the `[]` comes to over 80 chars.
```
[x] WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeOpenParenthesis
Warning: str_repeat(): Second argument has to be greater than or equal to 0 in /PEAR/PHP/CodeSniffer/Reports/Source.php on line 164

Call Stack:
    0.0000     127176   1. {main}() /phpcs:0
    0.0230    1029808   2. PHP_CodeSniffer_CLI->runphpcs() /phpcs:25
    0.0250    1069608   3. PHP_CodeSniffer_CLI->process() /PEAR/PHP/CodeSniffer/CLI.php:95
   71.3651   11686008   4. PHP_CodeSniffer_CLI->printErrorReport() /PEAR/PHP/CodeSniffer/CLI.php:873
   71.3781   11686856   5. PHP_CodeSniffer_Reporting->printReport() /PEAR/PHP/CodeSniffer/CLI.php:926
   71.3811   11703504   6. PHP_CodeSniffer_Reports_Source->generate() /PEAR/PHP/CodeSniffer/Reporting.php:258
   71.3811   11704080   7. str_repeat() /PEAR/PHP/CodeSniffer/Reports/Source.php:164

1
```

**Comment by westonruter** [comment]
This sounds like a bug in PHP_CodeSniffer itself. Maybe report the error there?

**Comment by jrfnl** [comment]
Looks like it's already being addressed ;-) https://github.com/squizlabs/PHP_CodeSniffer/commit/cd0c301ed5d61e0d02a8b57136469c4ec97cbcdc https://github.com/squizlabs/PHP_CodeSniffer/commit/cd0c301ed5d61e0d02a8b57136469c4ec97cbcdc

All the same, trying to have a combined name of max 70 chars seems like a good idea as 80 chars for these kind reports is quite normal.

**Comment by GaryJones** [comment]
PHPCS now supports a configured width of `auto` - no need to restrict yourself to 80 characters :-)
